Samples/NetworkMediaStreamer/RTSPServer/inc/RtspSession.h

```diff
@@ -0,0 +1,78 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+#pragma once
+
+// supported command types
+enum class RTSP_CMD
+{
+    UNKNOWN = 0,
+    OPTIONS,
+    DESCRIBE,
+    SETUP,
+    PLAY,
+    TEARDOWN
+};
+#define RTP_DEFAULT_PORT 54554
+#define RTSP_BUFFER_SIZE       10000    // for incoming requests, and outgoing responses
+#define RTSP_PARAM_STRING_MAX  200
+class RTSPSession

```

Comment:
>RTSPSession [](start = 6, length = 11)

Add Final

Reply:
i want people to be able to inherit and add functionality to RTSP server.

---
In reply to: [491108978](https://github.com/microsoft/Windows-Camera/pull/13#discussion_r491108978) [](ancestors = 491108978)